### DISCUSSION TITLE
Is there a way to set PATH variable when run/debug executable?

### DISCUSSION THREAD

**Discussion by quanzhuo** [microsoft/vscode-cmake-tools#3325] [Q&A] [answered]
I use vscode and this plugin on qt projects, everything works ok , except that I can not launch the built exeutable when click the Launch button in status bar. because the executable can not find qt library needed. 

I have to set PATH variable manually in the failed termimal, and run again to launch the program. 

```powershell
$env:PATH="$env:PATH;C:\Qt\5.15.2\msvc2019_64\bin"
```

Is there a way/setting to set PATH environment when run the built executable ?

**Comment by gcampbell-msft** [comment, marked as answer]
There are a couple of options here:

1. Open VS Code with your PATH variable already modified to include the items you need.
2. You should be able to create `cmake.debugConfig` in your settings.json and customize your configuration.
